.rubocop-bundler.yml

```diff
@@ -14,7 +14,7 @@ Lint/AssignmentInCondition:
   Enabled: false
 
 Lint/EndAlignment:
-  AlignWith: variable
+  EnforcedStyleAlignWith: variable
```

Comment:
Thanks for your review. I fixed it.
https://github.com/bundler/compact_index/pull/22/commits/9a0fec298a39753830e6c593b6afeb1674518ae5